Restore.cmd

```diff
@@ -1,2 +1,2 @@
 @echo off
-powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\build.ps1""" -restore %*"
+powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\build.ps1""" -restore  /p:RestoreUseStaticGraphEvaluation=true %*"
```

Comment:
This should be added here: https://github.com/dotnet/roslyn/blob/master/eng/build.ps1#L264
`*.cmd` files are meant to be just trivial stubs dispatching to build.ps1. No logic/settings should be specified here.

Reply:
thanks, I'll move it to Build.ps1